GVFS/GVFS.Hooks/Program.cs

```diff
@@ -112,7 +112,7 @@ private static void RemindUpgradeAvailable()
             int reminderFrequency = 10;
             int randomValue = random.Next(0, 100);
 
-            if (randomValue <= reminderFrequency && ProductUpgraderInfo.IsLocalUpgradeAvailable(GVFSHooksPlatform.GetInstallerExtension()))
+            if (randomValue <= reminderFrequency && ProductUpgraderInfo.IsLocalUpgradeAvailable(tracer: null))

```

Comment:
The hooks don't have a tracer instance?

Reply:
I will check again - but I did not see any tracers from the hooks. I don't recall us having any tracing from the hooks themselves for individual git commands...

Reply:
That's correct, the hooks don't have any tracing in them.